.devcontainer/scripts/container-creation.sh

```diff
@@ -5,10 +5,15 @@ set -e
 # GitHub doesn't pull in submodules by default when mounting
 # the repo directory on the container so we do this post-create
 git submodule update --init --recursive
+
 # Install SDK and tool dependencies before container starts
 # Also run the full restore on the repo so that go-to definition
 # and other language features will be available in C# files
 ./restore.sh
+
+# Add .NET Dev Certs to environment to facilitate debugging.
+dotnet dev-certs https
```

Comment:
cc/ @blowdart / @javiercn as this could be construed as a spicy change. We're auto-trusting the dev-certs but this is specifically for the GitHub Codespaces environment.

Reply:
Ah this is a per container command? So part of setup? Then sure but add a doc line saying to not do this in a public base image as saving a cert in an image would mean everything that bases itself off it would inherit that cert and any trust you grant on client machine.

Reply:
> Ah this is a per container command? So part of setup? 

Yes & yes.

https://github.com/dotnet/aspnetcore/blob/77599445aabd7bf357feb5cf8dfec7187148f1af/.devcontainer/devcontainer.json#L27

> Then sure but add a doc line 

https://github.com/dotnet/aspnetcore/pull/36000/files#r699783377